website/search/elastic_search.py

```diff
@@ -418,11 +416,10 @@ def serialize_contributors(node):
     return {
         'contributors': [
             {
-                'fullname': user.fullname,
-                'url': user.profile_url if user.is_active else None
-            } for user in node.visible_contributors
-            if user is not None
-            and user.is_active
+                'fullname': x['user__fullname'],
+                'url': '/{}/'.format(x['user__guids___id'])
+            } for x in
+            node.contributor_set.filter(visible=True, user__is_active=True).order_by('contributor___order').values('user__fullname', 'user__guids___id')
```

Comment:
Is this working? I just discovered that filtering on included relations doesn't quite work (https://github.com/chrisseto/django-include/issues/2).

```
In [1]: qs_include = Node.objects.filter(is_public=True).include('contributor__user')

In [2]: node = qs_include[0]

In [3]: node.contributor_set.all()
Out[3]: [<Contributor(user=saskia.vanderoord@ppw.kuleuven.be, read=True, write=True, admin=True, visible=True)>, <Contributor(user=tom.beckers@kuleuven.be, read=True, write=True, admin=True, visible=True)>]

In [4]: node.contributor_set.filter(visible=True)
Out[4]: []
```

If you wanted to keep the include, you could filter out in Python. It might be worth it.